client/src/Invitation.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
```

Comment:
For future reference, we can make this cleaning by combining the imports:
`import { Button, TextField, Dialog, etc } from "@material-ui/core`

Reply:
Got it!! Fixing it right now